### PR TITLE
update preseq/lcextrap errorStrategy to ignore

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: modules/nf-core/preseq/lcextrap/main.nf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 process PRESEQ_LCEXTRAP {
     tag "$meta.id"
     label 'process_single'
-    label 'error_retry'
+    label 'error_ignore'
```
Comment (edmundmiller) [inline modules/nf-core/preseq/lcextrap/main.nf:4]: Can we remove the label and make a custom one?

Retry it once, then if it fails the second time we ignore it?

Comment (sateeshperi) [reply modules/nf-core/preseq/lcextrap/main.nf:4]: `label 'error_retry_ignore'` ?

Comment (edmundmiller) [reply modules/nf-core/preseq/lcextrap/main.nf:4]: No just define the error strategy at that point 😂

Comment (SPPearce) [reply modules/nf-core/preseq/lcextrap/main.nf:4]: Retrying once and then failing sounds great, from what I recall this fails anytime the sample doesn't have enough depth.

**Pull request closed without merging**